src/mapeo-project.js

```diff
@@ -113,6 +119,15 @@ export class MapeoProject {
         db,
       }),
     }
+
+    this.#blobStore = new BlobStore({
+      coreManager: this.#coreManager,
+    })
+
+    this.$blobs = new BlobApi({
+      projectId: this.#projectId,
+      blobStore: this.#blobStore,
```

Comment:
Missing blobServer param - can be added as a constructor param for MapeoProject. ping @achou we'll need to create a BlobServer instance in MapeoClient.